File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F03.P2.java
import java.util.ArrayList;
import java.util.List;

class Main {
    static int probe() {
        List<Integer> xs = new ArrayList<>(List.of(7, 8, 9));
        // BEGIN PROBE F03.P2
        xs.set(1, 42);
        // END PROBE F03.P2
        return xs.get(1);
    }

    public static void main(String[] args) {
        System.out.println(probe());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F03.P2.go
package main

import "fmt"

func probe() int {
	xs := []int{7, 8, 9}
	// BEGIN PROBE F03.P2
	xs[1] = 42
	// END PROBE F03.P2
	return xs[1]
}

func main() {
	fmt.Println(probe())
}
